proposals/NNNN-manager-update-display-capability.md

```diff
@@ -0,0 +1,318 @@
+    SDLRegisterAppInterfaceResponse *response = (SDLRegisterAppInterfaceResponse *)notification.response;
+    if (!response.success.boolValue) { return; }
+    self.convertDeprecatedDisplayCapabilitiesNeeded = YES; // reset the flag
+    self.displays = [self createDisplayCapabilityListWithRegisterResponse:response];
+    :
+- (void)sdl_displayLayoutResponse:(SDLRPCResponseNotification *)notification {
+    SDLSetDisplayLayoutResponse *response = (SDLSetDisplayLayoutResponse *)notification.response;
+    if (!response.success.boolValue) { return; }
+    if (self.convertDeprecatedDisplayCapabilitiesNeeded) {
+        self.displays = [self createDisplayCapabilityListWithSetDisplayLayoutResponse:response];
+    }
+
+- (BOOL)sdl_saveSystemCapability:(SDLSystemCapability *)systemCapability completionHandler:(nullable SDLUpdateCapabilityHandler)handler {
+    :
+    } else if ([systemCapabilityType isEqualToEnum:SDLSystemCapabilityTypeDisplays]) {
+        self.convertDeprecatedDisplayCapabilitiesNeeded = NO;
+        [self sdl_updateCachedDisplayCapabilityList:systemCapability.displayCapabilities];
+    }
+}
+
+- (void)sdl_updateCachedDisplayCapabilityList(List<DisplayCapability> newCapabilities) {
+    // Similar to Java Code:
+    // 1. Copy the existing Array of existing window capabilities
+    // 2. Replace occurrences of new window capabilities in the copy
+    // 3. Apply the updated copy into the DisplayCapability
+    // 4. Store the new object in the `displays` property.
+    // 5. Convert objects into the properties `displayCapabilities`, `buttonCapabilities`, `softButtonCapabilities` and `presetBankCapabilities`
+}
+```
+
+## Potential downsides
+
+The downside of the DisplayCapabilities replacement is the fact that the developer has two different options of fetching them. Using deprecation flags and converting capabilities in both directions should be helpful to understand which API is the modern one. Existing apps may have issues with modern head units that don't provide old display capabilities anymore. These apps should be supported with the conversion but they need to do a library update.
+
+## Impact on existing code
+
+`DisplayCapabilities` is a widely used object which now gets deprecated. Compatibility is guaranteed with the object conversion. However applications and also the SDL libraries will face many deprecation notes by the IDE.
+
+The screen managers will be affected by the deprecations hence should upgrade to the new display capability. From a brief code review only private screen manager code is affected therefore the changes are considered as implementation details.
+
+## Alternatives considered
+
+An addition to the system capability mangers could be adding listeners for windows with a specified window ID instead of listening for all windows in a `DISPLAYS` notification. This would be helpful and convenient for screen managers and the application to be notified on specific window changes only.
```

Comment:
```suggestion
An addition to the system capability managers could be adding listeners for windows with a specified window ID instead of listening for all windows in a `DISPLAYS` notification. This would be helpful and convenient for screen managers and the application to be notified on specific window changes only.
```